src/taipy/core/config/data_node_config.py

```diff
@@ -38,8 +38,8 @@ class DataNodeConfig(Section):
             "in_memory".
             The default value is "pickle".
             Note that the "in_memory" value can only be used when `JobConfig^`.mode is "standalone".
-        scope (Scope^):  The `Scope^` of the data nodes instantiated from the data node config. The default value is
-            SCENARIO.
+        scope (Optional[Scope^]):  The optional `Scope^` of the data nodes instantiated from the data node config.
```

Comment:
The OCD in me feels weird looking at it haha. It's very very very minor, I'd ignore it if I were you 🤣 
```suggestion
        scope (Optional[Scope^]): The optional `Scope^` of the data nodes instantiated from the data node config.
```

Reply:
How can I miss this 😓 Need to fix this asap, or else taipy-core will break